spras/config/util.py

```diff
@@ -17,3 +26,18 @@ def _missing_(cls, value: Any):
                 if member.lower() == value:
                     return member
         return None
+
+# We also need to allow `CaseInsensitiveEnum` to be represented in yaml.safe_dump,
+# allowing us to safely log parameters in Snakemake:
+# https://github.com/yaml/pyyaml/issues/722#issue-1781352490
+yaml.SafeDumper.add_multi_representer(
+    CaseInsensitiveEnum,
+    yaml.representer.SafeRepresenter.represent_str,
+)
+
+class Empty(BaseModel):
+    """
+    The empty base model. Used for specifying that an algorithm takes no parameters,
+    yet are deterministic.
```

Comment:
```suggestion
    yet is deterministic.
```
Is this the correct phrasing?